packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -1608,10 +1598,25 @@ class SvgAttributes {
   /// The `mix-blend-mode` attribute.
   final BlendMode? blendMode;
 
+  /// The `font-family` attribute, as a string.
+  final String? fontFamily;
+
+  /// The font weight attribute.
+  final int? fontWeight;
```

Comment:
I think it's fine to have this as an int? but if we do we should probably get rid of the enum, otherwise it should be a FontWeight?

Reply:
done